doc/make/ubuntu-12.04-detailed.md

```diff
@@ -1,77 +0,0 @@
-Preparing working environment on Ubuntu 12.04
----------------------------------------------
-
-### Install basic packages
-
-    sudo apt-get install git
-    sudo apt-get install libgmp-dev
-    sudo add-apt-repository ppa:kalakris/cmake -y
-    sudo apt-get install cmake
-
-    sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
-    sudo update-alternatives --remove-all gcc
-    sudo update-alternatives --remove-all g++
-    sudo apt-get update
-    sudo apt-get install g++-4.8 -y
-    sudo apt-get upgrade -y && sudo apt-get dist-upgrade -y
-
-    sudo ln -s /usr/bin/g++-4.8 /usr/bin/g++
-
-### Optional packages
-
-    sudo apt-get install gitg
-    sudo apt-get install valgrind
-    sudo apt-get install doxygen
-    sudo apt-get install kcachegrind
-
-### Fork Lean on github : https://github.com/leanprover/lean
-
-### Create a projects directory
-
-    cd ~
-    mkdir projects
-    cd projects
-
-### Clone your fork
-
-    git clone https://github.com/[your-user-name]/lean.git
-
-### Build Lean in debug mode
-
-    cd lean
-    mkdir -p build/debug
-    cd build/debug
-    cmake -D CMAKE_BUILD_TYPE=Debug ../../src
-    make
-
-### If you are using Emacs, here are some basic configurations
```

Comment:
Should we preserve this section somewhere else?

Reply:
I don't think so.